lib/completion-manager.coffee

```diff
@@ -165,7 +165,7 @@ class CompletionManager extends LTool
     filedir = parsed_fname.dir
     filebase = parsed_fname.base  # name only includes the name (no dir, no ext)
 
-    labels = find_in_files(filedir, filebase, /\\label\{([^\}]+)\}/g)
+    labels = find_in_files(filedir, filebase, /\\label(?:\[[^\{\}]*\]){0,1}\{([^\}]+)\}/g)
```

Comment:
I think you should use `\[[^\]]*\]` instead of `\[[^\{\}]*\]`, in addition `?` may be better than `{0,1}`